spec/unit/recipes/test_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe 'jenkins_plugins::default' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new(platform: 'centos', version: '6.6').converge(described_recipe)
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.